CSNotificationView/CSNotificationView.m

```diff
@@ -383,6 +383,10 @@ - (void)dismissWithStyle:(CSNotificationViewStyle)style message:(NSString *)mess
 - (CGFloat)topLayoutGuideLengthCalculation
 {
     CGFloat top = MIN([UIApplication sharedApplication].statusBarFrame.size.height, [UIApplication sharedApplication].statusBarFrame.size.width);
+
+    UIView *view = self.parentNavigationController.view ?: self.parentViewController.view;
+    CGPoint origin = [view.superview convertPoint:view.frame.origin toView:[UIApplication sharedApplication].keyWindow.rootViewController.view];
```

Comment:
`convertPoint:` gets the origin of the `parent(Navigation)ViewController` inside of the `keyWindow`.
Using the `rootViewController` instead of the `keyWindow` directly, solves some problems with rotations. 

The `notificationView.origin.y` is the top of the `parentViewController` not the top of the `keyWindow`. So you should subtract the height of the status bar only if the origin of the `parentViewController` is actually the top of the `keyWindow`. Otherwise you have to calculate the height of the `notificationView` somehow differently. Perhaps there is a better way to achieve that.